src/libs/gasPrice/gasPrice.ts

```diff
@@ -198,11 +201,16 @@ export async function getGasPriceRecommendations(
     return { gasPrice: fee, blockGasLimit: lastBlock.gasLimit }
   }
   const prices = filterOutliers(txns.map((x) => x.gasPrice!).filter((x) => x > 0))
+
+  // use th fetched price as a min if not 0 as it could be actually lower
+  // than the hardcoded MIN.
+  const minOrFetchedGasPrice = BigInt(ethGasPrice) !== 0n ? BigInt(ethGasPrice) : MIN_GAS_PRICE
```

Comment:
what if eth_gasPrice returns 0, shouldn't we actually respect it? 

Also, is eth_gasPrice returning errors on networks that use 1559? If not, then let's not swallow the error and always log it, because it could lead to a network mysteriously not working without us knowing

if it returns 0 and we want to respect it, we need to distinct between a zero (0x00...000) and an empty response (0x)